br/pkg/restore/client.go

```diff
@@ -780,171 +549,6 @@ func (rc *Client) switchTiKVMode(ctx context.Context, mode import_sstpb.SwitchMo
 	return nil
 }
 
-// GoValidateChecksum forks a goroutine to validate checksum after restore.
-// it returns a channel fires a struct{} when all things get done.
-func (rc *Client) GoValidateChecksum(
```

Comment:
Restore to raw won't do the checksum now ? Then I think we will have a TODO to add it.

Reply:
yes, I haven't found the checksum process for RawKV. it should be considered in both backup and restore.

Reply:
yes, I haven't found the checksum process for RawKV. it should be considered in both backup and restore.